### PR TITLE
Add: OpenTTD 1.11.0 release post

### PR DESCRIPTION
**Pull request by TrueBrain**

### CODE REVIEW

**Comment by TrueBrain** [comment]
Reddit:
```
OpenTTD 1.11.0 just released!

Did you know you could plant tress in the Scenario Editor with a brush now? It is like painting, but with trees.

https://www.openttd.org/news/2021/04/01/openttd-1-11-0.html
```

Discord:
```
OpenTTD 1.11.0 just released!

Towns can now grow with tunnels too. Guess you won't be able to contain them any more!

https://www.openttd.org/news/2021/04/01/openttd-1-11-0.html
```

Twitter:
```
OpenTTD 1.11.0 just released!
Finally. I can build that 100% rainforest map I always wanted.
https://www.openttd.org/news/2021/04/01/openttd-1-11-0.html
```

tt-forusm:
```
OpenTTD 1.11.0 just released!
Don't hold tab too long; you might be in the wrong century now! (well, disable the fast-forward limiter first).
https://www.openttd.org/news/2021/04/01/openttd-1-11-0.html
```

Steam:
exactly the same post as on our website; need to find a proper image to go with it, will update when I get it.

File: _posts/2021-04-01-openttd-1-11-0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: OpenTTD 1.11.0
+author: TrueBrain
+---
+
+Another year, another release.
+Who would have guessed ;)
+
+With 1.11 we bring you a lot of new features which you hopefully enjoy as much as we did creating them:
+* Performance improvements - Fast-Forward is now so fast, there is a setting to rate limit it (on by default).
```
Comment (LordAro) [inline _posts/2021-04-01-openttd-1-11-0.md:10]: Worth mentioning OpenGL/"hardware acceleration" explicitly?

Comment (TrueBrain) [reply _posts/2021-04-01-openttd-1-11-0.md:10]: I wanted to mention a lot of things; but I couldn't find a funny way of wording it all. So I settled for this by the lack of better. We don't have to mention all the details, I guess.

Also, without giving more context, I was somewhat afraid people would think the game now has a 3D engine or something silly :P

But, if you have a single sentence to mention something like that, which isn't all too serious, please do make a suggestion :D

Comment (LordAro) [reply _posts/2021-04-01-openttd-1-11-0.md:10]: "In part due to a new hardware-accelerated video renderer, fast-forward is now..." ?

Comment (TrueBrain) [reply _posts/2021-04-01-openttd-1-11-0.md:10]: That would make a really long sentence. I think we just leave it for what it is now. The changelogs gives them the full story, I guess :P

**Review by LordAro — APPROVED** [review]

**Pull request merged**